File: Runtime/消息转发- 类方法/消息转发/BFBoy.m
//
//  BFBoy.m
//  动态方法解析
//
//  Created by WengHengcong on 2018/12/15.
//  Copyright © 2018 WengHengcong. All rights reserved.
//

#import "BFBoy.h"
#import <objc/runtime.h>

@implementation BFBoy

- (void)eat
{
    NSLog(@"%s" , __func__);
}

+ (void)eat
{
    NSLog(@"%s" , __func__);
}


@end

File: 编译/LLDBLearn/LLDBLearn/BFBoy.cpp
//
//  BFBoy.cpp
//  LLDBLearn
//
//  Created by 翁恒丛 on 2018/11/14.
//  Copyright © 2018年 翁恒丛. All rights reserved.
//

#include "BFBoy.hpp"

BFBoy::BFBoy(double a, bool handsome)
{
    this->asset = a;
    this->handsome = handsome;
};

void BFBoy::workingHard()
{
    asset = asset + 100;
}
